taxcalc/tests/test_reforms.py

```diff
@@ -27,12 +27,13 @@ def test_2017_law_reform(tests_path):
     with open(reform_file, 'r') as rfile:
         rtext = rfile.read()
     pol.implement_reform(Policy.read_json_reform(rtext))
+    # TODO: I'm unclear on how to handle the undefined warning behavior
+    # since the parameter is still defined in the JSON.
```

Comment:
I'm not sure how to handle the `CTC_c` parameter warning since it's still defined in `policy_current_law.json`, but according to this map, it's been redefined: 

https://github.com/PSLmodels/Tax-Calculator/blob/0e6dee1f5556b0da01a2141d39a77d8ca82b1146/taxcalc/policy.py#L65-L69

@MattHJensen  or @Peter-Metz  do you have any guidance for how this should be handled?

Reply:
Before TCJA, `CTC_c` represented the whole (non-refundable) credit and since TCJA, `CTC_c` represents the non-refundable portion of the credit. In my view, the warning message doesn't add much, especially since Tax-Calc 1.0.0 was released over a year ago.

cc @MattHJensen @hdoupe

Reply:
>  In my view, the warning message doesn't add much, especially since Tax-Calc 1.0.0 was released over a year ago.

Agreed that this warning can go given the time elapsed since the 1.0 release.